.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - { name: msrv, version: 1.68.0 }
+          - { name: msrv, version: 1.70.0 }
```

Comment:
I have an action for keeping this in sync manually if you're okay using it: https://github.com/actions-rust-lang/msrv?tab=readme-ov-file#usage